modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchIndexingImpl.java

```diff
@@ -290,7 +290,7 @@ void processIndexCommand(IndexingCommand cmd) {
             return;
         }
         if (log.isDebugEnabled()) {
-            logDebugMessageTruncated(String.format("Index request: curl -XPUT 'http://localhost:9200/%s/%s/%s' -d '%s'",
+            logDebugMessageTruncated(String.format("Index request: curl -XPUT 'http://localhost:9200/%s/%s' -d '%s'",
```

Comment:
to be clear, this should be in the same commit as https://github.com/nuxeo/nuxeo/pull/4293/commits/f2afb3d1cc87dd9b40c7e2d50499c7af381dd55e#diff-f0d6271e8fca3291a638cf9797c75ddaL295

Reply:
will do that better next time